docs/suite.md

```diff
@@ -6,8 +6,8 @@ You can form suites using separate tests, groups, and modules.
 
 Each suite must be defined in the `<VendorName>/<ModuleName>/Test/Mftf/Suite` directory.
 
-The tests for each suite are generated in a separate directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/_generated/`.
-All tests that are not within a suite are generated in the _default_ suite at `.../Magento/FunctionalTest/_generated/default/`.
+The tests for each suite are generated in a separate directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/_generated/`.
+All tests that are not within a suite are generated in the _default_ suite at `.../Magento/_generated/default/`.
```

Comment:
Let's match the path with what's in line 9.

Reply:
done

Reply:
I mean change it to `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/_generated/default/`

Reply:
yes, that's what I did : )